marketplace_docs/templates/Fabric/fabfile.py

```diff
@@ -57,6 +57,8 @@ def install_files():
             put(lpath,rpath,mirror_local_mode=True)
 
 
+def wait_for_apt_dpkg_lock_release():
+    run("while fuser /var/lib/dpkg/lock /var/lib/apt/lists/lock /var/cache/apt/archives/lock >/dev/null 2>&1; do echo 'Waiting for apt/dpkg lock release' sleep 5; done; sudo apt-get -yy update")
```

Comment:
Do we need the `apt upgrade` in this function?  I think given its name, we probably just this function to do the waiting part, and the `apt upgrade` to be a separate thing done afterwards.

Reply:
Yeah. That should not be there.